src/screens/SignupScreen.tsx

```diff
@@ -29,9 +31,9 @@ export const SignupScreen: React.FC<SignupProps> = () => {
     const [error, setError] = React.useState<string | null>(null)
     const [isLoading, setIsLoading] = React.useState(false)
     const [isPasswordVisible, setIsPasswordVisible] = React.useState(false)
-    const [userRole, setUserRole] = React.useState<string | null>(null)
+    const [userRole, setUserRole] = React.useState<UserOption | null>(UserOption.MEMBER)
```

Comment:
did you want to automatically check the radio button for member as soon as the user loads into the signup page

Reply:
Yes, however, if you feel that giving them a different UI that makes them choose would be best I'm down with that 😄